components/youtube_video/youtube_video.tsx

```diff
@@ -84,13 +84,17 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
                         <ExternalImage src={image?.secure_url || image?.url || ''}>
                             {(safeUrl) => (
                                 <img
+                                    style={{cursor: 'pointer'}}
                                     src={safeUrl}
                                     alt='youtube video thumbnail'
                                     className='video-thumbnail'
```

Comment:
can this style go inside of _videos.scss? instead of styles tag

Reply:
Sure